README.md

```diff
@@ -49,229 +49,253 @@ path = "https://allencell.s3.amazonaws.com/aics/nuc-morph-dataset/hipsc_fov_nucl
 image = BioImage(path)
 print(image.get_image_dask_data())
 ```
-## OME Zarr V2 Writer
 
-Import the writer and utilities:
+## Writing OME-Zarr Stores
 
+The `OMEZarrWriter` can write **both** Zarr v2 (NGFF 0.4) and Zarr v3 (NGFF 0.5) formats.
+
+### basic writer example (2D YX)
 ```python
-from bioio_ome_zarr.writers import (
-    OmeZarrWriterV2,
-    chunk_size_from_memory_target,
-    compute_level_shapes,
-    compute_level_chunk_sizes_zslice,
-    resize,
-)
+from bioio_ome_zarr.writers import OMEZarrWriter
+import numpy as np
 
-```
+# Minimal 2D example (Y, X)
+data = np.random.randint(0, 255, size=(64, 64), dtype=np.uint8)
 
-Use utility functions directly:
+writer = OMEZarrWriter(
+    store="basic.zarr",
```

Comment:
so "store" is the directory that will contain all the files for the zarr?  Maybe make the value 
store="./basic.zarr"?